src/spikeinterface/core/core_tools.py

```diff
@@ -312,6 +275,43 @@ def write_binary_recording(
     executor.run()
 
 
+# used by write_binary_recording + ChunkRecordingExecutor
```

Comment:
I moved this so it follows the pattern that all the `ChunkRecordingExector` chunk functions follow. The function is after the caller of `ChunkRecordingExecutor`. Sorry for my OCD.

Reply:
No worries! Honestly I also found it aannoying